src/PantherTestCase.php

```diff
@@ -23,6 +26,39 @@ abstract class PantherTestCase extends WebTestCase
 
         public const CHROME = 'chrome';
         public const FIREFOX = 'firefox';
+        public const COVERAGE_DIRECTORY = __DIR__.'/../var/panther-coverage';
+
+        public function run(TestResult $result = null): TestResult
+        {
+            $result = parent::run($result);
+
+            if (!is_dir(self::COVERAGE_DIRECTORY)) {
+                return $result;
+            }
+
+            if (!class_exists(Filesystem::class)) {
+                throw new \LogicException('The Filesystem component is not installed. Try running "composer require --dev symfony/filesystem".');
+            }
+
+            if (!class_exists(Finder::class)) {
+                throw new \LogicException('The Finder component is not installed. Try running "composer require --dev symfony/finder".');
+            }
+
+            /** @var \SplFileInfo[] $files */
+            $files = (new Finder())->in(self::COVERAGE_DIRECTORY)->files()->name('*.code_coverage');
+            foreach ($files as $file) {
+                $content = file_get_contents($file->getRealPath());
```

Comment:
The Finder component's [`getIterator()` method](https://github.com/symfony/symfony/blob/337426c43ae0d22f4bafaf9cbf8849dea1310c57/src/Symfony/Component/Finder/Finder.php#L611) actually returns an instance of [`Symfony\Component\Finder\SplFileInfo`](https://github.com/symfony/symfony/blob/337426c43ae0d22f4bafaf9cbf8849dea1310c57/src/Symfony/Component/Finder/SplFileInfo.php#L19) which has a [`getContents` method](https://github.com/symfony/symfony/blob/337426c43ae0d22f4bafaf9cbf8849dea1310c57/src/Symfony/Component/Finder/SplFileInfo.php#L74), maybe it would make sense to utilize it instead:

```suggestion
            $files = (new Finder())->in(self::COVERAGE_DIRECTORY)->files()->name('*.code_coverage');
            foreach ($files as $file) {
                $content = $file->getContents();
```

Reply:
You’re right, I’ll change it 😀